### ISSUE TITLE
Move pytorch implementation under a pytorch namespace

### ISSUE THREAD

**Issue by jlema** [microsoft/BackwardCompatibilityML#99]
Now that we have a tensorflow namespace, for consistency I'd like to suggest that we also have a pytorch namespace and move the current loss and respective helpers there. This will require updating all examples, and other docs, and it's a breaking change so it will require a major version change.
